service-api/src/main/java/greencity/dto/shoppinglistitem/ShoppingListItemRequestDto.java

```diff
@@ -7,12 +7,14 @@
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
+import lombok.experimental.SuperBuilder;
 
 @NoArgsConstructor
 @AllArgsConstructor
 @Getter
 @Setter
 @EqualsAndHashCode
+@SuperBuilder
```

Comment:
Why do we need it?
This class does not extend any other one

Reply:
Why the default value of `id` is `0` if you use @Min(1)
Based on [@Min documentation](https://docs.oracle.com/javaee/6/api/javax/validation/constraints/Min.html): 
`The annotated element must be a number whose value must be higher or equal to the specified minimum.`

Reply:
> Why do we need it? This class does not extend any other one

Because the documentation says that all classes in the hierarchy must be annotated
![image](https://user-images.githubusercontent.com/74297669/214841677-90167e94-88bc-43b1-a38d-7b1327f40840.png)
I tried to remove it, but i had an error
![image](https://user-images.githubusercontent.com/74297669/214842002-5f75a43f-99f8-47d7-bb3e-911271cdf604.png)